bentoml/utils/__init__.py

```diff
@@ -55,8 +55,18 @@ def isidentifier(s):
         return re.match(r"[A-Za-z_][A-Za-z_0-9]*\Z", s) is not None
 
 
+def _import_ruamel_yaml_module():
```

Comment:
recommend refactoring - 

add a new file `bentoml/utils/ruamel_yaml.py`

```python
try:
        from ruamel.yaml import *
except ModuleNotFoundError:
        from ruamel_yaml import *
```

In other files you can simply do:
```python
from bentoml.utils.ruamel_yaml import YAML
```